### PR TITLE
BMP180 Sensor implementation

### PR DESCRIPTION
**Pull request by BarajasPaul**
source code for bmp180 sensor was was developed, and the following subtask were made:
*create an install file for the sensor
*source code to get values from sensor
*test cases to validate its functionality

### CODE REVIEW

File: tests/sensors_test.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+# -*- encoding: utf-8 -*-
+
+import unittest
+from DataQuatic.sensors import BMP180
+
+class TestStringMethods(unittest.TestCase):
```
Comment (ericknoriega) [inline tests/sensors_test.py]: Are we testing Strings or Sensors?

**Review by ericknoriega — DISMISSED** [review]

File: tests/sensors_test.py (hunk 1 above)
Comment (BarajasPaul) [reply tests/sensors_test.py]: sensors, let met fix it

**Comment by BarajasPaul** [comment]
fixed it

**Review by gmgalvan — APPROVED** [review]
all good Pol

**Pull request merged**